README.md

```diff
@@ -1 +1,15 @@
-# docver-sdk-go
\ No newline at end of file
+# docver-sdk-go
+
+Document verification go client for API dataspike.io.
+
+Installation:
+```sh
+go get github.com/dataspike-io/docver-sdk-go
+```
+
+## Quick start
+
+```
+dc := dataspike.NewDataspikeClient(http.DefaultClient, dataspike.WithSandbox(), dataspike.WithToken("your token"))
```

Comment:
Lets make default client "default", without requirement to specify it explicitly